code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -36,6 +36,65 @@
 	projectile_type = /obj/item/projectile/beam
 	charge_cost = 50
 
+/obj/item/weapon/gun/energy/laser/failure_check(var/mob/living/carbon/human/M)
+	if(damaged)
+		if(projectile_type == "/obj/item/projectile/beam/captain" && prob(5))
```

Comment:
This projectile should only be fired by the captain's laser,which doesn't appear in a lockbox.